io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,112 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon (Test at every major update)
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # SSH Agent access
+  - --socket=ssh-auth
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=home
```

Comment:
then xdg-documents.

or `persist`.

Reply:
[I originally did](https://github.com/ShobuPrime/flathub/commit/826ce86bc219e317216ff93fd5fc4da2e6fbc4a6) use `xdg-documents`, but the [bot didn't like that](https://buildbot.flathub.org/#/builders/37/builds/3299)

Reply:
You just need to add it to [this list](https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json)

Reply:
neither.

The bot is fine with `xdg-documents`. its `xdg-data` and `xdg-config` it doesn't like